packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -1282,6 +1286,16 @@ export default class TradeStore extends BaseStore {
         }
     }
 
+    get is_dtrader_v2_enabled() {
+        const is_dtrader_v2 = JSON.parse(localStorage.getItem('FeatureFlagsStore') ?? '').data.dtrader_v2;
```

Comment:
We have isDTraderV2 function in shared. It checked localstorage and  innerWidth. Can it be reused? Or modified?

Reply:
im seeing instances where pathnames are checked individually

`isDTraderV2() && !is_contract_details `

OR

```
if (is_positions)
        return <PositionsLoader initial_app_loading={initial_app_loading} is_closed_tab={is_closed_tab} />;
if (is_contract_details) return <ContractDetailsLoader />;
```

so if i am to use the the shared function then 

where i use this.is_dtrader_v2_enabled it'll be replaced with

`isDtraderV2()  && (window.location.pathname.startsWith(routes.trade) || window.location.pathname.startsWith('/contract/'))`

in two separate places.

Reply:
slightly messy.

Reply:
at some point we'll likely need some kind of cleanup since this kind of logic with many variations is used in many places.

Reply:
Okay-okay, no problem)